.travis.yml

```diff
@@ -53,6 +53,7 @@ cache:
     - ~/pmd-bin-5.4.1
     - ~/bakalint-0.4.0
     - ~/dart-sdk/bin
+    - ~/.local/tailor/
```

Comment:
ya. good catch.  it worked *once* without it because ~/.local/bin is in the PATH , and tailor's install.sh drops a sym link in there.